.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,123 @@
+name: 🐛 Bug Report
+description: Report a bug that affects Lit or a Lit library/tool
+
+body:
+  - type: dropdown
+    id: affected
+    validations:
+      required: true
+    attributes:
+      label: Which package(s) are affected?
+      multiple: true
+      description: Note you can select more than one.
```

Comment:
Maybe it's obvious but should we say something like "If other, mention in description below"? Eh, maybe it'll just be extra noise.

Reply:
Done